packages/Charts/src/demos/demo.ts

```diff
@@ -0,0 +1,71 @@
+import * as DG from 'datagrok-api/dg';
+import * as grok from 'datagrok-api/grok';
+
+
+export class Demo {
+  static async chordViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('ChordViewer');
+  }
+
+  static async globeViewerDemo() {
+    const df = await grok.data.getDemoTable('geo/earthquakes.csv');
+    const tableView = grok.shell.addTableView(df);
+    // semantic type detection is needed for the latitude, longitude and magnitude columns
+    DG.debounce(df.onSemanticTypeDetected, 50).subscribe((_) => tableView.addViewer('GlobeViewer'));
+  }
+
+  static async groupAnalysisViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('GroupAnalysisViewer');
+  }
+
+  static async radarViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('RadarViewer');
+  }
+
+  static async sankeyViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SankeyViewer');
+  }
+
+  static async sunburstViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SunburstViewer');
+  }
+
+  static async surfacePlotDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
```

Comment:
This dataset makes no sense for the surface plot. I suggest to programmatically create a good one, such as 3d parabolic function or something fancy like that. Perhaps @AnnaMuza already has a good case for that.

Reply:
done